packages/docs/src/components/page-templates/SearchPage.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { TextField } from '@cmsgov/design-system';
+import { useFlexSearch } from 'react-use-flexsearch';
+import { useStaticQuery, graphql } from 'gatsby';
+import { SearchDataStore, SearchQuery } from '../../helpers/graphQLTypes';
+import avoidRefresh from '../../helpers/avoidRefresh';
+
+const SearchPage = () => {
```

Comment:
Can all this be done in the `pages/search.tsx` file? It seems kind of weird that we'd have to define it in `pages`, `content` and `page-templates`

Reply:
Yeah probably, I had set it up this way as was the way things have been wired, at some point I tried moving it all into one file and ran into some issue so I didn't look at it again, will see about making that happen.   The `SearchForm` in content is just a search form component so that will remain separate for use anywhere.